cli/client_test.go

```diff
@@ -47,17 +50,24 @@ func setup(
 	asc apppb.AppServiceClient,
 	dataClient datapb.DataServiceClient,
 	buildClient buildpb.BuildServiceClient,
-	defaultFlags *map[string]string,
+	defaultFlags *map[string]any,
```

Comment:
Had to change this to `*map[string]any` to allow for an integer-typed `count` flag to be specified in test setup.